models/experimental/functional_unet/tt/model_preprocessing.py

```diff
@@ -63,12 +63,12 @@ def create_unet_model_parameters(model: unet_shallow_torch.UNet, input_tensor: t
         "num_cores_nhw": 55,
     }
 
-    parameters.c1["conv_blocking_and_parallelization_config_override"] = {"act_block_h": 16 * 32}
+    parameters.c1["conv_blocking_and_parallelization_config_override"] = {"act_block_div": 15}
```

Comment:
why were values here changed? 16->15

Reply:
It was 16 tiles before, i.e. 512. Before act_block_h would specify the size of the exact size of the act_block. `act_block_div` specifies how many times the largest possible act_block must be divided. 

`largest_act_block_h = act_block_h * act_block_div`

Reply:
IMO, this approach makes it easier to choose a more performant value.

Reply:
Maybe I'm missing something but how can a model writer assume the height of the block?